### PR TITLE
Xcontent deletion op 113

### PR DESCRIPTION
**Pull request by Shree05Ram**

### CODE REVIEW

File: devmgmtV2/controllers/filemgmt.controller.js [hunk 1]
```diff
@@ -248,6 +248,20 @@ let deleteFileFromDisk = (req, res) => {
   
   if (ext === '.ecar') {
     cmd = `rm -rf ${file}* ${json_dir}* ${xcontent}*`;
+     //count the no of files in ecar_files's dir
+    fs.readdir(dir, (err, files) => {
+      let ecar_count = files.length - 1;
+      if (ecar_count === 0) {
+        let XC = path.resolve(dir, '..', 'xcontent');
+        fs.readdir(XC, (err, files) => {
+          if (err) throw err;
```
Comment (chetansk) [inline devmgmtV2/controllers/filemgmt.controller.js]: Throw an error and do what? I mean,  checking error is mandatory here ?

File: devmgmtV2/controllers/filemgmt.controller.js [hunk 2]
```diff
@@ -248,6 +248,20 @@ let deleteFileFromDisk = (req, res) => {
   
   if (ext === '.ecar') {
     cmd = `rm -rf ${file}* ${json_dir}* ${xcontent}*`;
+     //count the no of files in ecar_files's dir
+    fs.readdir(dir, (err, files) => {
+      let ecar_count = files.length - 1;
+      if (ecar_count === 0) {
+        let XC = path.resolve(dir, '..', 'xcontent');
+        fs.readdir(XC, (err, files) => {
```
Comment (chetansk) [inline devmgmtV2/controllers/filemgmt.controller.js]: Is this code unit tested?

File: devmgmtV2/controllers/filemgmt.controller.js (hunk 1 above)
Comment (Shree05Ram) [reply devmgmtV2/controllers/filemgmt.controller.js]: Checking error is not mandatory. If case any error occurs, that could be caught.

File: devmgmtV2/controllers/filemgmt.controller.js (hunk 2 above)
Comment (Shree05Ram) [reply devmgmtV2/controllers/filemgmt.controller.js]: yes.

**Pull request merged**